grakn-graph/src/main/java/ai/grakn/factory/GraknSessionImpl.java

```diff
@@ -99,8 +102,10 @@ public GraknComputer getGraphComputer() {
 
     @Override
     public void close() throws GraphRuntimeException {
-        checkClosure(openGraphTxs(), graph);
-        checkClosure(openGraphBatchTxs(), graphBatch);
+        int openTransactions = openTransactions(graph) + openTransactions(graphBatch);
+        if(openTransactions > 1){
```

Comment:
That should be 0. You are correct. I was confused by TinkerGraph because that will always have an open transaction.